src/app/static/src/components/parameters/Phases.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="p-2">
+    <div class="phase-block-container">
+      <div
+        v-for="block in phaseBlocks"
+        :key="block.index"
+        class="phase-block"
+        :class="phaseClassFromIndex(block.index)"
+        :style="`height:${block.height}; width:${block.width}; left: ${block.left}`"
+      ></div>
+    </div>
+    <div class="phase-block-base"></div>
+    <div v-for="phase in displayPhases"
+         :key="phase.index"
+         class="phase-description p-2 mt-2"
+         :class="phaseClassFromIndex(phase.index)">
+      <div class="phase-header">
+        <span class="font-weight-bold">Phase {{phase.index}}</span> ({{phase.days}} days)
+      </div>
+      <div class="phase-dates mb-3">{{phase.start}} - {{phase.end}}</div>
+      <div class="phase-rt">Rt: <span class="font-weight-bold">{{phase.value}}</span></div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "@vue/composition-api";
+import { Rt } from "@/types";
+import dayjs, { Dayjs } from "dayjs";
+import duration from "dayjs/plugin/duration";
+
+dayjs.extend(duration);
+
+interface Props {
+    phases: Array<Rt>,
```

Comment:
We seem to be using both `Array<Rt>` and `Rt[]` formats. Latter marginally easier to read but I'm not picky.

Reply:
OK, let's standardise on `[]`